discord/ui/item.py

```diff
@@ -83,7 +83,7 @@ def to_component_dict(self) -> Dict[str, Any]:
     def refresh_component(self, component: Component) -> None:
         return None
 
-    def refresh_state(self, interaction: Interaction) -> None:
+    def refresh_state(self, data) -> None:
```

Comment:
I don't understand why this change was made

Reply:
yeah, it'll be always an interaction

Reply:
This was part of adjusting the `refresh_state` methods across items to take a dict instead of `Interaction`, as this is how it was handled in InputText and the modal `dispatch` method; I figured this made more sense than changing modal dispatch to use an `Interaction` for `refresh_state`.
![image](https://user-images.githubusercontent.com/41271523/173477951-2f2df60b-b20d-4a22-a0f7-bb5f9434c8ca.png)

Reply:
Alright